.ldrelease/update-version.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -e
+
 VERSION_GO=internal/version/version.go
 VERSION_GO_TEMP=${VERSION_GO}.tmp
 sed "s/const Version =.*/const Version = \"${LD_RELEASE_VERSION}\"/g" ${VERSION_GO} > ${VERSION_GO_TEMP}
```

Comment:
Should we add a comment here to remind us to invoke `update-bitbucket-metadata.sh`?

```sh
# TODO: update bitbucket metadata when techops approves the ci bot
# https://launchdarkly.atlassian.net/servicedesk/customer/portal/11/TO-5131
# ./update-bucket-metadata.sh
```